scvid/module/incremental_pca.py

```diff
@@ -121,3 +121,11 @@ def components_kg(self) -> torch.Tensor:
         parallel to its eigenvectors. The components are sorted by decreasing ``explained_variance_k``.
         """
         return self.V_kg
+
+    def predict(self, x_ng: torch.Tensor) -> torch.Tensor:
+        r"""
+        Embedding of the input data ``x_ng`` into the principal component space.
+        """
+        if self.transform is not None:
+            x_ng = self.transform(x_ng)
+        return x_ng @ self.V_kg.T
```

Comment:
What about mean subtraction?

Reply:
Mean subtraction will just shift all the vectors by a constant vector, right? I thought in that case it shouldn't have any effect on vector similarity search.